websites/N/Neko-sama.fr/iframe.ts

```diff
@@ -0,0 +1,12 @@
+const iframe = new iFrame();
+
+iframe.on("UpdateData", async () => {
+	if (document.querySelector("video") === null) return;
+
+	const video = document.querySelector("video");
```

Comment:
Use optional chaining

Reply:
> Use optional chaining

Can I have a little help please? I don't know how to concatainate return and variable

Reply:
I suppose Encrypted ask you to remove the if statement and add `?.` for each attribute of the video element sent with `iframe.send()`

Reply:
```ts
const iframe = new iFrame();

iframe.on("UpdateData", async () => {
	const video = document.querySelector("video");
	iframe.send({
		time: video?.currentTime,
		duration: video?.duration,
		paused: video?.paused,
	});
});
```

Like that ?

Reply:
Yup

Reply:
Okay, I'll commit that